Openverse/DSA-Website-feature-ui-enhancements/src/components/HomePage.jsx

```
@@ -0,0 +1,240 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FaGithub, FaRegLightbulb, FaListUl, FaLayerGroup, FaTree, FaProjectDiagram, FaSitemap, FaSortNumericDown, FaChartLine, FaCode, FaRandom, FaSlidersH } from 'react-icons/fa';
+import './HomePage.css';
+import Robocats from '../assets/Robocats.svg';
+import logo from '../assets/openverse2.svg';
+
+const Card = ({ title, link, icon, type, enabled = true, color }) => {
```

ESLint

[error] 'type' is defined but never used.

(no-unused-vars)